packages/core/src/dialects/mysql/query.js

```diff
@@ -20,11 +20,22 @@ export class MySqlQuery extends AbstractQuery {
   }
 
   async run(sql, parameters) {
-    this.sql = sql;
     const { connection, options } = this;
+    const shouldMinifyAliases = (parameters || {}).shouldMinifyAlias;
 
     const showWarnings = this.sequelize.options.showWarnings || options.showWarnings;
 
+    if ((this.sequelize.options.minifyAliases || shouldMinifyAliases) && this.options.includeAliases) {
+      for (const [alias, original] of _.toPairs(this.options.includeAliases)
+        .sort((a, b) => b[1].length - a[1].length)) {
+        const reg = new RegExp(_.escapeRegExp(original), 'g');
```

Comment:
Thanks a lot for such an amazing explanation Zoe! Will remove that bunch and post this will start working on putting this up for other dialects.

Reply:
Maybe we should add a TODO linking back to this comment so we don't forget to fix it in a separate PR?

Reply:
Awesome will put it, also I think I can pick up fixing all these issues too! It would also give more context on the repo and possibly raise even better PRs in the future! 😅

Reply:
I think we can even open an issue about it